malpolon/data/datasets/poverty_dataset.py

```diff
@@ -0,0 +1,170 @@
+import os
+import json
+from typing import Callable
+
+import numpy as np
+import rasterio
+import pandas as pd
+from matplotlib import pyplot
+
+import torch
+from torch.utils.data import Dataset
+from torch.utils.data import DataLoader
+from torchvision import transforms
+
+from malpolon.data.data_module import BaseDataModule
+
+import datetime
+
+
+class PovertyDataModule(BaseDataModule):
+    def __init__(
+            self,
+            tif_dir: str = 'landsat_tiles/',
+            dataset_path: str = 'examples/poverty/dataset/',
+            labels_name: str = 'observations_2013+.csv',
+            train_batch_size: int = 32,
+            inference_batch_size: int = 16,
+            num_workers: int = 8,
+            **kwargs
+    ):
+
+        """DataModule for the Poverty dataset.
+        Args:
+            tif_dir (str): directory containing the tif files
+            dataset_path (str): path to the dataset
+            labels_name (str): name of the csv file containing the labels
+            train_batch_size (int): batch size for training
+            inference_batch_size (int): batch size for inference
+            num_workers (int): number of workers for the DataLoader
+            fold (int): fold to use for training
+            transform (torchvision.transforms): transform to apply to the data"""
+
+        super().__init__()
+        dataframe = pd.read_csv(dataset_path + labels_name)
+        self.dataframe = dataframe
+        self.dataframe_train = dataframe[dataframe['subset'] == 'train']
+        self.dataframe_val = dataframe[dataframe['subset'] == 'val']
+        self.dataframe_test = dataframe[dataframe['subset'] == 'test']
+        self.tif_dir = dataset_path + tif_dir
```

Comment:
Consider encapsulating your strings in `pathlib.Path` objects to avoid '/' (slash) sensitiveness.
e.g.: 
```
from pathlib import Path
self.tif_dir = Path(dataset_path) + Path(tif_dir)
```

Reply:
As we discussed, I removed the dataset from the PR.
I did make the chage though, thanks for the advice!